activities.yml

```diff
@@ -0,0 +1,2798 @@
+<link>'s imagesrcset and imagesizes attributes:
```

Comment:
I'm constantly amazed at YAML and its ability to interpret what would be garbage input in other contexts.  I assume that this is machine-generated.

Reply:
It is yes (`json2yaml.py`)